### PR TITLE
Add drafts for courses A and B to `is_k1`

### PR DESCRIPTION
**Pull request by Hamms**
so they can benefit from K1-only features like TTS

### CODE REVIEW

File: dashboard/app/models/script.rb [hunk 1]
```diff
@@ -380,7 +380,7 @@ def self.beta?(name)
 
   # TODO(asher): Rename this method to k1?, removing the need to disable lint.
   def is_k1?  # rubocop:disable PredicateName
-    name == 'course1'
+    ['course1', 'coursea-draft', 'courseb-draft'].include?(name)
```
Comment (ashercodeorg) [inline dashboard/app/models/script.rb]: nit: Can we reference CONSTANTS (from somewhere?) rather than strings?

Comment (balderdash) [reply dashboard/app/models/script.rb]: They're in lib/cdo/script_constants.rb if you need 'em.

Comment (Hamms) [reply dashboard/app/models/script.rb]: so convenient! Done

File: dashboard/app/models/script.rb [hunk 2]
```diff
@@ -380,7 +380,11 @@ def self.beta?(name)
 
   # TODO(asher): Rename this method to k1?, removing the need to disable lint.
   def is_k1?  # rubocop:disable PredicateName
-    name == 'course1'
+    [
+      Script::COURSEA_DRAFT_NAME,
+      Script::COURSEB_DRAFT_NAME,
+      Script::COURSE1_NAME
+    ].include?(name)
```
Comment (Bjvanminnen) [inline dashboard/app/models/script.rb:387]: Is there any reason k1 isn't a property on individual scripts, instead of doing it this way?

Comment (Hamms) [reply dashboard/app/models/script.rb:387]: Particularly given that we do precisely that for levels, I agree that that would be better. I'll look into doing that in a separate PR

**Comment by Hamms** [comment]
PTAL

**Pull request merged**